### PR TITLE
#167749941 Integrate HoundCI

### PR DESCRIPTION
**Pull request by Naimaoye**
#### What does this PR do?
Integrate houndci into the project
#### Description of task to be completed?
Edit `.hound.yml`
#### How should this be manually tested?
By raising a PR
#### Relevant pivotal tracker stories?
[167749941](https://www.pivotaltracker.com/n/projects/167749941)
#### Questions:
What are the styles we want to configure? I mean the content of .hound.yml file

### CODE REVIEW

**Comment by daviddamilola** [comment]
it looks good, how do we test?

**Comment by Naimaoye** [comment]
we can test by creating a pull request

**Comment by alatos2** [comment]
It looks perfect. I already made a pull request and I saw **Hound — No violations found. Woof!**
hound

Kindly note that your PR naming convention should follow this format - #(story-id) story description.

**Review by hustlaviola — COMMENTED** [review]
Awesome. You can, however, edit the PR comment though. Here is a screenshot of the area that needs editing
Screenshot (37)

**Comment by Naimaoye** [comment]
okay guys, thanks for the feedbacks. I will work on it

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+coffeescript:
```
Comment (hustlaviola) [inline .hound.yml]: Good job. To make this cleaner, only eslint config is needed, we can do without the rest.

**Review by ldonjibson — COMMENTED** [review]
Good job! Weldone

**Comment by davidteejay** [comment]
Well done!!

**Review by davidteejay — APPROVED** [review]

**Review by hustlaviola — APPROVED** [review]

**Review by alatos2 — APPROVED** [review]

**Review by ascii-dev — APPROVED** [review]
LGTM

**Review by ldonjibson — COMMENTED** [review]
Great work!

**Pull request merged**